src/floris/simulation/floris.py

```diff
@@ -12,84 +12,154 @@
 
 # See https://floris.readthedocs.io for documentation
 
+from __future__ import annotations
 
 import json
+import pickle
+from pathlib import Path
 
-import floris.logging_manager as logging_manager
-
-from floris.simulation import Farm
-from floris.simulation import TurbineGrid
-from floris.simulation import sequential_solver
+import attr
+import yaml
 
-# from .wake import Wake
-from .turbine import Turbine
-from .flow_field import FlowField
-from .wake_velocity.jensen import JensenVelocityDeficit
+import floris.logging_manager as logging_manager
+from floris.utilities import FromDictMixin
+from floris.simulation import (
+    Farm,
+    Wake,
+    Turbine,
+    FlowField,
+    TurbineGrid,
+    sequential_solver,
+)
+from floris.simulation.wake_velocity import CurlVelocityDeficit, JensenVelocityDeficit
+from floris.simulation.wake_deflection import JimenezVelocityDeflection
+
+
+MODEL_MAP = {
+    # "wake_combination": {"""The Combination Models"""},
+    "wake_deflection": {"jimenez": JimenezVelocityDeflection},
+    # "wake_turbulence": {"""The Turbulence Models"""},
+    "wake_velocity": {"curl": CurlVelocityDeficit, "jensen": JensenVelocityDeficit},
+}
+VALID_WAKE_MODELS = [
```

Comment:
Is this used?

Reply:
This was supposed to be removed, thanks for the catch!